src/fu-util.c

```diff
@@ -1646,16 +1647,37 @@ fu_util_get_updates (FuUtilPrivate *priv, gchar **values, GError **error)
 			continue;
 		}
 		child = g_node_append_data (root, dev);
+		upgrade_count++;
 
 		/* add all releases */
 		for (guint j = 0; j < rels->len; j++) {
 			FwupdRelease *rel = g_ptr_array_index (rels, j);
 			g_node_append_data (child, g_object_ref (rel));
 		}
 	}
-
-	if (g_node_n_nodes (root, G_TRAVERSE_ALL) > 1)
-		fu_util_print_tree (root, title);
+	if (log_output == NULL) {
+		if (g_node_n_nodes (root, G_TRAVERSE_ALL) > 1)
+			fu_util_print_tree (root, title);
+	} else if (upgrade_count == 0) {
+		return TRUE;
+	} else {
+		g_autofree gchar *upgrade_str = NULL;
+		if (upgrade_count == 1) {
+			/* TRANSLATORS: message letting the user know upgrades are available
+			* %1 is a positive integer */
+			upgrade_str = g_strdup_printf (_("%u device has a firmware upgrade available. "
+							 "Run `fwupdmgr get-upgrades` for more information"),
+							 upgrade_count);
+		} else {
+			/* TRANSLATORS: message letting the user know upgrades are available
+			* %1 is a positive integer */
+			upgrade_str = g_strdup_printf (_("%u devices have firmware upgrades available. "
+							 "Run `fwupdmgr get-upgrades` for more information"),
+							 upgrade_count);
+		}
```

Comment:
This smells to me like another command, e.g. like `get-summary-updates` or something. Perhaps it even belongs in fwupdagent instead, and then we can parse the JSON in the MOTD generator.

Reply:
Adjusted it.